anaconda_project/internal/pip_api.py

```diff
@@ -87,27 +87,10 @@ def installed(prefix):
     if not os.path.isdir(prefix):
         return dict()
 
-    # In pip 9, there's a big ugly deprecation warning by default if
```

Comment:
By replacing `pip list` with `pip freeze` we avoid a ton of painful logic figuring out what `--format` keyword argument to use. In fact, the current version of this code is broken with newer versions of pip, because `--format=legacy` is gone.